substratevm/src/com.oracle.svm.jni/src/com/oracle/svm/jni/hosted/JNINativeCallWrapperSubstitutionProcessor.java

```diff
@@ -37,17 +41,30 @@
  */
 class JNINativeCallWrapperSubstitutionProcessor extends SubstitutionProcessor {
     private final Map<ResolvedJavaMethod, JNINativeCallWrapperMethod> callWrappers = new ConcurrentHashMap<>();
+    private final DuringSetupAccessImpl access;
+
+    JNINativeCallWrapperSubstitutionProcessor(DuringSetupAccessImpl access) {
+        super();
```

Comment:
Please get rid of this redundant `super()`.

Reply:
Done